app/redux/wallet/actions.js

```diff
@@ -199,6 +249,16 @@ export const setTransactions = ({ transactions }: { transactions: TxList }): Act
   payload: { transactions }
 });
 
+export const setContacts = ({ contacts }: { contacts: Contact[] }): Action => ({
+  type: SET_CONTACTS,
+  payload: { contacts }
+});
+
+export const setLastUsedAddresses = ({ lastUsedAddresses }: { lastUsedAddresses: Contact[] }): Action => ({
```

Comment:
add TODO
this should be changed to addLastUsedAddress later on

Reply:
ok